### ISSUE TITLE
services_log uses syslog handler

### ISSUE THREAD

**Issue by pelme** [pytest-dev/pytest-services#20]
services_log configures a syslog handler by default. This (sometimes) leads to interesting crashes like 

```
   File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/logging/handlers.py", line 936, in emit
     self.socket.send(msg)
 AttributeError: 'SysLogHandler' object has no attribute 'socket'
```

full exception here: https://gist.github.com/pelme/99e3ed217f1fe432fa7f5b18d341742d

Can we simply get rid of the `SyslogHandler` creation in https://github.com/pytest-dev/pytest-services/blob/47e5ae4833338063e9c431db8999e2e928f4d2fe/pytest_services/log.py#L14-L20 ? And just return a plain logger and leave it to the user to configure the logging system?

**Comment by jaraco** [comment]
The exception I frequently see is in an atexit handler:

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/logging/__init__.py", line 2112, in shutdown
    h.close()
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/logging/handlers.py", line 892, in close
    self.socket.close()
AttributeError: 'SysLogHandler' object has no attribute 'socket'
```

**Comment by jaraco** [comment]
Filed [this ticket upstream](https://bugs.python.org/issue38780) for the atexit error.

**Comment by jaraco** [comment]
A workaround was released in 2.0.1. Feel free to open a separate ticket to capture an enhancement request for improving the logging fixture.
